.eslintrc

```diff
@@ -51,9 +66,11 @@
       }
     ],
     "@typescript-eslint/no-unused-vars": "warn",
+    "@typescript-eslint/ban-ts-comment": "warn",
     "padding-line-between-statements": [
       "error",
       { "blankLine": "always", "prev": "*", "next": "return" }
-    ]
+    ],
+    "unused-imports/no-unused-imports-ts": 2
```

Comment:
2 stands for "error", Maybe that should be used

"unused-imports/no-unused-imports-ts": "error"